.space.kts

```diff
@@ -1,28 +1,64 @@
-job("KInference / Build and Test") {
-    host("Build and test") {
-        env["AWS_ACCESS_KEY"] = Secrets("aws_access_key")
-        env["AWS_SECRET_KEY"] = Secrets("aws_secret_key")
+val jsContainer = "registry.jetbrains.team/p/ki/containers-ci/ci-corretto-17-firefox:1.0.1"
+val jvmContainer = "amazoncorretto:17"
 
-        shellScript("Install Firefox, xvfb and JDK") {
-            content = """
-                apt-get update && apt-get install firefox xvfb openjdk-17-jdk -y -f
-            """.trimIndent()
+job("KInference / Build") {
+    container("Build With Gradle", jvmContainer) {
+        kotlinScript { api ->
+            api.gradlew("assemble", "--parallel", "--console=plain", "--no-daemon")
         }
+    }
+}
+
+job("KInference / Test / JVM") {
+    container("JVM Tests", jvmContainer) {
+        kotlinScript { api ->
+            api.gradlew("jvmTest", "--parallel", "--console=plain", "-Pci", "--no-daemon")
```

Comment:
Is there any actual reason to use that instead of shell scripts?

Reply:
No, but why not? :)